example/lib/screens/forms/auto_suggest_box.dart

```diff
@@ -35,8 +41,10 @@ class _AutoSuggestBoxPageState extends State<AutoSuggestBoxPage>
         subtitle(content: const Text('A basic AutoSuggestBox')),
         CardHighlight(
           child: Row(children: [
+            // Either Expanded widget or just hard coded size like this
             SizedBox(
-              width: 350.0,
+              // width: 350.0,
+              width: smallScreenWidth ? 250 : 350.0,
```

Comment:
I would say to use a `ConstrainedBox` with the `maxWidth` set to `350`

Reply:
I've tried this approach, but it didn't work as expected. No responsiveness.
If you have a better solution just give me a hint.